corehq/apps/hqcase/tests/test_loadtest_users.py

```diff
@@ -22,23 +31,25 @@ class LoadtestUserTest(TestCase, DomainSubscriptionMixin):
     @classmethod
     def setUpClass(cls):
         super(LoadtestUserTest, cls).setUpClass()
-        cls.domain = create_domain(DOMAIN)
```

Comment:
what is the rationale behind this change? If something else isn't cleaning up after itself, can we change that instead?